### PR TITLE
May 22 Fixes

### PR DESCRIPTION
**Pull request by dshanske**
Refreshed the dependencies a bit.

### CODE REVIEW

**Comment by dshanske** [comment]
@pfefferle I reverted the previous merge as part of this PR if you want to look. Mostly refreshing the icons...which I need to look at in future in more detail, as that set is started to get too big.

File: templates/h-card.php [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+<div class="hcard-display h-card vcard u-author">
 	<div class="hcard-header">
 		<?php if ( ! $avatar ) { ?>
-			<a class="u-url url fn u-uid" href="<?php echo esc_url( $url ); ?>"rel="author">
+			<a class="u-url url fn u-uid" href="<?php echo esc_url( $url ); ?>">
 				<p class="hcard-name p-name n"><?php echo $name; ?></p>
 			</a>
 		<?php } else { ?>
-			<a class="u-url url fn u-uid" href=" <?php echo esc_url( $url ); ?>" rel= "author"><?php echo $avatar; ?></a>
+			<a class="u-url url fn u-uid" href=" <?php echo esc_url( $url ); ?>"><?php echo $avatar; ?></a>
```
Comment (pfefferle) [inline templates/h-card.php]: can you also remove the blank after the `href="`?

**Review by pfefferle — APPROVED** [review]
LGTM. But maybe we can change the loading logic for the icons css a bit, so that it only loads if it is used.

**Comment by dshanske** [comment]
> LGTM. But maybe we can change the loading logic for the icons css a bit, so that it only loads if it is used.

I want to work on that in a future version. The icon set I'm using has tripled...which is great, but a lot of them don't need to be loaded.

**Pull request merged**